desc/compute/_neoclassical.py

```diff
@@ -272,3 +278,253 @@ def eps_32(data):
+    # ∂G/∂((λB₀)⁻¹) =     λ²B₀  ∫ dℓ (1 − λ|B|/2) / √(1 − λ|B|) ∂|B|/∂ψ / |B|
+    # ∂V/∂((λB₀)⁻¹) = 3/2 λ²B₀  ∫ dℓ √(1 − λ|B|) K / |B|
+    # ∂g/∂((λB₀)⁻¹) =     λ²B₀² ∫ dℓ (1 − λ|B|/2) / √(1 − λ|B|) |∇ψ| κ_g / |B|
+    # tan(π/2 γ_c) =
+    #              ∫ dℓ (1 − λ|B|/2) / √(1 − λ|B|) |∇ρ| κ_g / |B|
+    #              ----------------------------------------------
+    # (|∇ρ| ‖e_α|ρ,ϕ‖)ᵢ ∫ dℓ [ (1 − λ|B|/2)/√(1 − λ|B|) ∂|B|/∂ψ + √(1 − λ|B|) K ] / |B|
+
+    def d_v_tau(data, B, pitch):
+        return safediv(2.0, jnp.sqrt(jnp.abs(1 - pitch * B)))
+
+    def drift1(data, B, pitch):
+        return (
+            safediv(1 - 0.5 * pitch * B, jnp.sqrt(jnp.abs(1 - pitch * B)))
+            * data["|grad(rho)|*kappa_g"]
+            / B
+        )
+
+    def drift2(data, B, pitch):
+        return (
+            safediv(1 - 0.5 * pitch * B, jnp.sqrt(jnp.abs(1 - pitch * B)))
+            * data["|B|_psi|v,p"]
+            / B
+        )
+
+    def drift3(data, B, pitch):
+        return jnp.sqrt(jnp.abs(1 - pitch * B)) * data["K"] / B
+
+    def Gamma_c(data):
+        """∫ dλ ∑ⱼ [v τ γ_c²]ⱼ."""
+        # Note v τ = 4λ⁻²B₀⁻¹ ∂I/∂((λB₀)⁻¹) where v is the particle velocity,
+        # τ is the bounce time, and I is defined in Nemov eq. 36.
+        bounce = Bounce1D(grid, data, quad, automorphism=None, is_reshaped=True)
+        points = bounce.points(data["pitch_inv"], num_well=num_well)
+        v_tau, f1, f2 = bounce.integrate(
+            [d_v_tau, drift1, drift2],
+            data["pitch_inv"],
+            data,
+            ["|grad(rho)|*kappa_g", "|B|_psi|v,p"],
+            points=points,
+            batch=batch,
+        )
+        gamma_c = jnp.arctan(
+            safediv(
+                f1,
+                (
+                    f2
+                    + bounce.integrate(
+                        drift3,
+                        data["pitch_inv"],
+                        data,
+                        "K",
+                        points=points,
+                        batch=batch,
+                        quad=quad2,
+                    )
+                )
+                * bounce.interp_to_argmin(data["|grad(rho)|*|e_alpha|r,p|"], points),
+            )
+        )
+        return (4 / jnp.pi**2) * (
+            (v_tau * gamma_c**2).sum(axis=-1)
+            * data["pitch_inv"] ** (-2)
+            * data["pitch_inv weight"]
+        ).sum(axis=-1)
+
+    # We rewrite equivalents of Nemov et al.'s expression's using single-valued
+    # maps of a physical coordinates. This avoids the computational issues of
+    # multivalued maps. It further enables use of more efficient methods, such as
+    # fast transforms and fixed computational grids throughout optimization, which
+    # are used in the ``Bounce2D`` operator on a developer branch. Also, Nemov
+    # assumes B^ϕ > 0 in some comments; this is not true in DESC, but the
+    # computations done here are invariant to the sign.
```

Comment:
I remove statements about other's assumptions in the fourier bounce branch to be more profssional